index.html

```diff
@@ -1,12 +1,270 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Девайс</title>
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>HTML Academy: Девайс</title>
+</head>
+
+<body>
+
+<header class="site-header">
+
+  <nav class="user-navigation">
+    <img class="logo" src="img\logo.svg" alt="Логотип Device">
+    <h1 class="visually-hidden">Device</h1>
+    <form class="user-navigation search-form" action="https://echo.htmlacademy.ru/" method="POST">
+      <div class="user-navigation search-item">
+        <img class="user-navigation search-item seacrh-ico" alt="Поиск">
```

Comment:
```suggestion
        <img class="user-navigation search-item search-ico" alt="" src="">
```

src - обязательный атрибут элемента img, так что надо его писать, даже если пустой,
и орфографическая ошибка в слове search - тебе надо поставить проверку орфографии (у тебя ВС кода? там есть такое, русскую и английскую проверку орфографии)

Reply:
не сделано